gravitl/netclient
# feat(GRA-1081): add hosts settings page

TESTING STEPS:
1. Open NCGUI
2. Go to Settings from the File menu
3. Check that the host settings are correct
